qt/python/mantidqt/widgets/instrumentview/instrument_view.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
```

Comment:
Could we maybe consider calling this file `api` or something. As we already have a `view` file it may be clear what the intention of this file is.

Could we also add the standard licence agreement to the top?

Reply:
The file has been renamed to api, and the standard licnece agreement is added to the top